requirements.txt

```diff
@@ -41,3 +41,4 @@ retry~=0.9.2                  # wmcore,wmagent,wmagent-devtools,reqmgr2,reqmon,g
 rucio-clients~=1.25.5         # wmcore,wmagent,global-workqueue,reqmgr2ms
 Sphinx~=4.0.3                 # wmcore,wmagent,wmagent-devtools,reqmgr2,reqmon,acdcserver,global-workqueue,reqmgr2ms
 SQLAlchemy~=1.3.3             # wmcore,wmagent
+pyjwt~=2.3.0                  # wmagent,wmagent-devtools,reqmgr2,reqmon,acdcserver,global-workqueue,reqmgr2ms,wmcore
```

Comment:
Luckily I just looked this up in Pypi and noticed that this name isn't correctly written. As far as I know, Pypi is case sensitive, so please update it to `PyJWT`, from: https://pypi.org/project/PyJWT/

Reply:
done